### PR TITLE
studio:  polish existing prop components

### PR DESCRIPTION
**Pull request by achen5671**
## Summary
https://tsu.atlassian.net/browse/IR-2521

This PR polishes existing components to match the figma design. There are more components that needs to be cleaned up in this section but I wanted to break it out to a separate PR since those component need to be migrated in its entirety using tailwind i.e `Post Processing`, `Media Setting`, `Scene Setting`, etc. This PR also already touches 23 files so it might be better to break it up

This PR polish
`Camera`, `Transform`, `EnvMap Bake`, `Scene Preview Camera`, `Volumetric`, etc

## References
closes #_insert number here_

### CODE REVIEW

File: packages/ui/src/components/editor/input/Vector3/index.tsx [hunk 1]
```diff
@@ -124,13 +124,13 @@ export const Vector3Input = ({
   const vz = value.z
 
   return (
-    <div className="flex flex-wrap justify-start gap-1.5">
+    <div className="flex flex-row flex-nowrap justify-start gap-1.5">
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/input/Vector3/index.tsx]: why was `flex-nowrap` added?
on smaller screens the vector input is overflowing
image

File: packages/ui/src/primitives/tailwind/Select/index.tsx [hunk 2]
```diff
@@ -144,10 +144,11 @@ const Select = <T extends OptionValueType>({
         endComponent={
           <MdOutlineKeyboardArrowDown
             size="1.5em"
-            className={`text-theme-primary transition-transform ${showOptions.value ? 'rotate-180' : ''}`}
+            className={`text-[#444444] transition-transform ${showOptions.value ? 'rotate-180' : ''}`}
             onClick={toggleDropdown}
           />
         }
+        containerClassname="rounded-md"
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Select/index.tsx]: why did we add border radius here?
[in designs](https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi/iR-Engine-Design-Library-File?node-id=2508-3527&t=FC7FlAaAzLUoVy6o-0), select primitive does not have a border radius

File: packages/ui/src/primitives/tailwind/Input/index.tsx [hunk 3]
```diff
@@ -97,7 +97,7 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
     }
 
     const twcontainerClassName = twMerge(
-      'flex w-full flex-col items-center ',
+      'flex flex-col items-center overflow-hidden',
```
Comment (aditya-mitra) [inline packages/ui/src/primitives/tailwind/Input/index.tsx]: There are some distortions like the focus in the admin panel. Primitive behaviour like full width needs to be changed with every component using it.
image

File: packages/ui/src/components/editor/properties/envMapBake/index.tsx [hunk 4]
```diff
@@ -162,7 +162,9 @@ export const EnvMapBakeNodeEditor = (props) => {
       description="For Adding EnvMap bake in your scene"
       icon={<IoMapOutline />}
     >
-      <Button onClick={() => uploadBPCEMBakeToServer(props.entity)}>Bake</Button>
+      <Button className="my-1 ml-auto mr-6 px-10" onClick={() => uploadBPCEMBakeToServer(props.entity)}>
+        Bake
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/properties/envMapBake/index.tsx]: could you add this to translation file?

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
Great work on the detailing 🔥 They look nicer now!
A few amendments like translations and reverting the changes to primitives would get this PR ready to merge.

File: packages/ui/src/components/editor/input/Vector3/index.tsx (hunk 1 above)
Comment (achen5671) [reply packages/ui/src/components/editor/input/Vector3/index.tsx]: resolved

File: packages/ui/src/primitives/tailwind/Select/index.tsx (hunk 2 above)
Comment (achen5671) [reply packages/ui/src/primitives/tailwind/Select/index.tsx]: oh I didnt realized we had this file for primitive. will resolve

**Review by aditya-mitra — APPROVED** [review]

**Pull request merged**